### PR TITLE
DAT-1122 django2 compatibility

### PR DESCRIPTION
**Pull request by tmeehanpw**
### Description

This PR makes this package compatible with django2.

Monolith Test pass in the monolith testing PR (read there for more notes): https://github.com/paymentworks/monolith/pull/4271

#### Jira Tracking

https://paymentworks.atlassian.net/browse/DAT-1122

#### Screenshots (If Applicable)
N/A

#### Describe Rollback Scenarios
Revert

#### What are some risks involved if the code in this PR is released?
This package is the cornerstone of updates / edit registrations.

#### Scenarios Manually Tested on Load Environments
✅ Create an Update
✅ Tests pass on monolith (django1)

#### Migration Notes/Highlights
N/A

### CODE REVIEW

File: django_model_changes/changes.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def current_state(self):
 
             # Foreign fields require special care because we don't want to trigger a database query when the field is
             # not yet cached.
-            if field.rel:
```
Comment (tmeehanpw) [inline django_model_changes/changes.py:119]: main change

**Review by isibley765 — APPROVED** [review]
Glad you got this working 👌 the commit being present but still getting the same error is baffling, but a version change being required makes sense on a separate thread

File: setup.py [hunk 2]
```diff
@@ -8,7 +8,7 @@
 
 setup(
     name='django-model-changes-py3',
-    version='0.15.1',
```
Comment (tmeehanpw) [inline setup.py:11]: This version bump necessary so the cached package isn't used during pip install.

**Review by eugenetrigubapw — APPROVED** [review]
Nice work on testing this 👍 I'm presuming this would still require a monolith deploy to fully release to ensure the latest requirements.txt is installed which has the latest version of this package?

**Comment by awgreenblatt** [comment]
Nice! Glad we're getting this updated.

**Comment by tmeehanpw** [comment]
> Nice work on testing this 👍 I'm presuming this would still require a monolith deploy to fully release to ensure the latest requirements.txt is installed which has the latest version of this package?

Yes a monolith re-deploy would be necessary for it to get the latest package version. It is probably worth doing a redeploy just to verify that there isn't any unexpected behavior changes.

**Pull request merged**